packages/manager/apps/web/client/app/hosting/hosting.html

```diff
@@ -99,7 +99,7 @@ <h1 class="d-inline-block mb-0 word-break">
                 data-ng-if="$ctrl.tabMenu"
                 data-text="{{:: $ctrl.tabMenu.title }}">
                 <oui-header-tabs-item
-                    data-ng-repeat-start="item in $ctrl.tabMenu.items track by $index"
+                    data-ng-repeat-start="item in $ctrl.tabMenu.items"
```

Comment:
Maybe we could track by target ?

```suggestion
                    data-ng-repeat-start="item in $ctrl.tabMenu.items track by item.target"
```

Reply:
yes you're right i though it was an array of strings too but it's not